### ISSUE TITLE
Can we reproduce the result of paper with the default settings？

### ISSUE THREAD

**Issue by yanghu819** [chufengt/iccv19_attribute#7]

**Comment by chufengt** [comment]
Tune the hyper-parameters subtly can obtain the results.
Or, you can try some other tricks, like warm-up or extra loss weights.

**Comment by yanghu819** [comment]
spcifically, what is the exact hyper-parameters config of peta?

**Comment by yanghu819** [comment]
![result](https://user-images.githubusercontent.com/44990984/80386842-8131e400-88da-11ea-9329-7682a17993ba.jpg)

**Comment by happyprotean** [comment]
@kickingass One possible reason for the gap between your result and the paper is that you choose different 35 attributes to train and test. The paper choose 35 attributes which positive ratio is large 5%, but you just choose first 35 attributes.

**Comment by yanghu819** [comment]
可以给出具体的设置吗，或者pretrained weights，要不没法复现出您的结果
